pages/overview/visualizations/active_drifting_contributors.py

```diff
@@ -30,10 +30,11 @@
                     [
                         dbc.PopoverHeader("Graph Info:"),
                         dbc.PopoverBody(
-                            "Select the intervals that make sense for your community!\n\
-                            <ACTIVE> contributors that have a contribution within the drifting parameter.\n\
```

Comment:
Feel like we need some version on this either in the description or info page

Reply:
The definition is on the Info page in the other PR.